src/transformation/visitors/errors.ts

```diff
@@ -44,31 +44,38 @@ export const transformTryStatement: FunctionVisitor<ts.TryStatement> = (statemen
         const catchParameter = statement.catchClause.variableDeclaration
             ? transformIdentifier(context, statement.catchClause.variableDeclaration.name as ts.Identifier)
             : undefined;
-        const catchParameters = () => (catchParameter ? [lua.cloneIdentifier(catchParameter)] : []);
```

Comment:
What happened here, this seems like the change from the other branch? Guess you need to merge master into this branch.

Reply:
Yeah, somehow those changes made it here.
I just force pushed a new commit without those changes